doc/src/base/base.md

```diff
@@ -61,6 +61,23 @@ Finally:
 and `outer` is parsed as a keyword when used to modify the scope of a variable in an iteration specification of a `for` loop.
 Creation of variables named `where`, `in`, `isa` or `outer` is allowed though.
 
+Unlike `in`, `isa` and `outer`, `where` remains being an operator when used in special operations as stated above
```

Comment:
`where` is never an operator. It's sometimes syntax, but there's never a situation where the surface syntax calls `where(x,y)`.

Reply:
Hmm, @JeffBezanson proposed that particular language, so maybe he disagrees with me on what it means to be an operator. Let's have him chime in here. I would have just said "is parsed infix" without calling it an operator.

Reply:
I think an "infix operator" is fine since it makes no sense to just say "infix".

"infix" on what? lol

Reply:
`infix` would be an adverb in that instance, modifying `parsed`.

Reply:
I kinda see `where` as an operator that supports just only the "infix" syntax. It doesn't matter for it to be `where(x, y)` valid, before its considered an "operator", afterall the `+(x, y)` is akin to Julia.